recipe_scrapers/costco.py

```diff
@@ -0,0 +1,43 @@
+# mypy: allow-untyped-defs
+
+from ._abstract import AbstractScraper
+from ._utils import normalize_string
+
+
+class Costco(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "costco.com"
+
+    def author(self):
+        return "Costco Connection"
+
+    def title(self):
+        return self.soup.find("h1", style="font-size: 2.5em;").get_text()
+
+    def ingredients(self):
+        parent_div = self.soup.find("div", {"class": "col-xs-12 col-md-5"})
+        if parent_div:
+            parent_ul = parent_div.find("ul")
+            if parent_ul:
+                ingredients_list_items = parent_ul.find_all("li")
+                ingredients_list = [
+                    normalize_string(li.get_text()) for li in ingredients_list_items
+                ]
+                return ingredients_list
+
+    def instructions(self):
+        parent_div = self.soup.find("div", {"class": "col-xs-12 col-md-5"})
+        if parent_div:
+            instructions_paragraphs = parent_div.find_all("p")
+            instructions_text = "/n".join(
```

Comment:
```suggestion
            instructions_text = "\n".join(
```

This should use the escape character (`\`) instead of a forward-slash (`/`).

Reply:
(if only we had test assertions to check that `instructions_list` returned multiple values at test runtime.. :smile:)

Reply:
This explains why I had an unbelievably hard time getting the test to pass. Should be all set now! (i hope)

Reply:
Hehe.  Yep, that looks good to me - thanks!